### PR TITLE
Print output of cluster bash script to both terminal and file

### PR DESCRIPTION
**Pull request by Stefan-Ethernal**
# Description

This PR changes the helper cluster script to print the output of both `stdout` and `stderr` to both terminal and file for each validator node.

**Note:** would be nice to maybe group it somehow to a folder because currently files are written to the root folder of the project and it gets overwritten if cluster script is run multiple times.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [ ] I have tested this code manually

### CODE REVIEW

**Review by vcastellm — COMMENTED** [review]
Why this change? I'm not against it but also not much in favor, search in terminal works well

**Comment by Stefan-Ethernal** [comment]
> Why this change? I'm not against it but also not much in favor, search in terminal works well

I can think of at least two reasons at a glance:
1. Terminal output has limited capacity (at least on VS Code Ubuntu) and if we are running cluster, it gets clipped away pretty fast, which makes searching inconvenient,
2. Logs from different nodes are interleaved, which makes it hard to follow if we want to track execution on a specific node.

So this way the best of both approaches so depending on use case, one could opt to search through the files or the terminal itself.

**Comment by vcastellm** [comment]
> > Why this change? I'm not against it but also not much in favor, search in terminal works well
> 
> I can think of at least two reasons at a glance:
> 
> 1. Terminal output has limited capacity (at least on VS Code Ubuntu) and if we are running cluster, it gets clipped away pretty fast, which makes searching inconvenient,

You can change that in settings in majority of terminals
image

> 2. Logs from different nodes are interleaved, which makes it hard to follow if we want to track execution on a specific node.
> 

For this I prefer to identify each node with a prefix + color similar to docker compose, if this is complex I'm ok with this.

> So this way the best of both approaches so depending on use case, one could opt to search through the files or the terminal itself.

**Comment by Stefan-Ethernal** [comment]
> You can change that in settings in majority of terminals

Yes, but still I've experienced that during running a cluster if I was about to search Terminal content at the same time, it would start highlighting wrong words (believe this is because of terminal content constantly being changed) and experienced it at least on Ubuntu (haven't tried on Mac OS though) which makes debugging harder. 

So I'd still would like to keep option to output logs to the files and make it optional (https://github.com/0xPolygon/polygon-edge/pull/1282/commits/eb57954ed46334d31ffa215baf8cc094d1d70dd7; not sure if we can make this if/else to have less repetitions, by for example extracting common parameters to local variable(s)).

> For this I prefer to identify each node with a prefix + color similar to docker compose, if this is complex I'm ok with this.

Not sure how to accomplish this, to be honest with you.

**Review by vcastellm — APPROVED** [review]

**Review by b00ris — APPROVED** [review]

**Review by ferranbt — APPROVED** [review]
